staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go

```diff
@@ -63,10 +63,10 @@ const (
 )
 
 var supportedValidationReason = sets.NewString(
-	string(field.ErrorTypeInvalid),
-	string(field.ErrorTypeForbidden),
-	string(field.ErrorTypeRequired),
-	string(field.ErrorTypeDuplicate),
+	string(apiextensions.FieldValueRequired),
```

Comment:
`FieldValueRequired` is in the list twice, this should be `FieldValueInvalid`

I'm surprised this passed tests

Reply:
Changed and added test case for this.